include/tao/pegtl/require_apply.hpp

```diff
@@ -4,14 +4,6 @@
 #ifndef TAO_PEGTL_REQUIRE_APPLY_HPP
 #define TAO_PEGTL_REQUIRE_APPLY_HPP
 
-#include <utility>
-
-#include "apply_mode.hpp"
-#include "config.hpp"
```

Comment:
This include is needed for `TAO_PEGTL_NAMESPACE`, likewise for `require_apply0.hpp`